src/wrap-store/wrapStore.js

```diff
@@ -109,6 +111,7 @@ export default (store, {
   /**
    * Setup extended external connection 
    */
-  chrome.runtime.onConnectExternal.addListener(connectState);
-
+  if (chrome.runtime.onConnectExternal) {
```

Comment:
It would be user friendly if an `else` clause could be added here with a console.log to notify Firefox users that a feature (which one, I'm not sure... @tshaddix, you should know) from `react-chrome-redux` is unavailable.